src/routes/api/users.js

```diff
@@ -1,6 +1,13 @@
-import Users from '../../controllers/users';
+import express from 'express';
+import Users from '../../controllers/user.controller';
 
-export default (server) => {
-  /* Users Routes Here */
-  server.post('/api/v1/auth/signup', Users.signup);
-};
+const router = express.Router();
+
+router.get('/users', (req, res) => {
+  res.status(200).json({
+    message: 'The Users Router is working ...'
+  });
+});
+router.post('/users/auth/signup', Users.signup);
```

Comment:
I think this goes against the agreed route. It should be `api/v1/auth/signup`

Reply:
There is already a `versioning process` in application's entry point. It is very bad that if i should want to change the version to v2, then I must start to tamper with the files themselves; when all you need to do is change it via the entry point for all files and modules.
Check `index.js` for the versioning

Reply:
I'm not talking about versioning. I'm talking about the `/users/`.
Maybe I didn't make it clear. 

The `/users/` shouldn't be there. When consuming this endpoint, we should make a call to `/api/v1/auth/signup` and not `/api/v1/users/auth/signup`. The team agreed on the former.

Reply:
Okay, I didn't actually get you the first time, but now I did. Of course, I get it, it somehow `slipped`. Fixing it now.